haystack/backends/elasticsearch5_backend.py

```diff
@@ -272,6 +272,23 @@ def build_search_kwargs(
         if dwithin is not None:
             filters.append(self._build_search_query_dwithin(dwithin))
 
+        if limit_to_registered_models is None:
```

Comment:
This stuff was just missing altogether from the django-haystack master branch. I noticed it because there was a rogue record without a `django_ct` field in the QA index and haystack was blowing up blindly accessing `_source['django_ct']`